CHANGES.md

```diff
@@ -4,10 +4,52 @@
 
 ### Breaking Changes
 
-* `XMonad.Util.NamedScratchpad`:
+  * `XMonad.Util.NamedScratchpad`:
 
   - Deprecated the module; use `XMonad.Util.NamedScratchpad` instead.
```

Comment:
What's the difference between `XMonad.Util.NamedScratchpad` and `XMonad.Util.NamedScratchpad`?

```
1. XMonad.Util.NamedScratchpad
2. XMonad.Util.NamedScratchpad
```

Don't see any :-(

Reply:
Pretty sure that was intended to be `X.U.Scratchpad`; see https://github.com/xmonad/xmonad-contrib/issues/756.

Reply:
Oh, okay

Reply:
Ah yes, silly typo there.  Thanks for pushing a fix already :)